docs/infrastructure.md

```diff
@@ -611,3 +580,79 @@ and click "Try it out" and then "Execute" to execute a query.
+query your new local neurobagel node,
+you have two options:
+
+### As part of local federation
+Use this option if you:
+
+- already have deployed other local neurobagel nodes 
+that you want your users to query alongside the new node
+- want your users to be able to query 
+all public neurobagel nodes together with your new node
+- plan on adding more local neurobagel nodes in the
+near future
+
+In this case, skip directly to the instructions on
+setting up [local query federation](federate.md).
+
+### As a standalone service
+Use this option if you
+
+- plan on only deploying a single node
+- want your users to only search data
+in the new node you deployed
+
+In this case, you need to deploy the query tool
+as a standalone docker container.
+
+```bash
+docker run -d -p 3000:3000 --env API_QUERY_URL=http://localhost:8000/ --name query_tool neurobagel/query_tool:latest
+```
+
+Make sure to replace the value of `API_QUERY_URL` with the `IP:PORT` or domain name of the
+new neurobagel node-API you just deployed!
+
+If using the default port mappings, 
+you can reach your local query tool at [http://localhost:3000](http://localhost:3000) once it is running.
+
+To see the exact configuration that your new docker
+container is running with (e.g. for debugging),
+you can run
+
+```bash
+docker inspect query_tool
+```
+
+??? note "Note on using a graphical query tool to send API requests"
+    The `NB_API_ALLOWED_ORIGINS` variable defaults to an empty string (`""`) when unset, meaning that your deployed API will only be accessible via direct `curl` requests to the URL where the API is hosted (see [this section](#test-the-new-deployment) for an example `curl` request).
+
+    However, in many cases you may want to make the API accessible by a frontend tool such as our [browser query tool](https://github.com/neurobagel/query-tool).
+    To do so, you must explicitly specify the origin(s) for the frontend using `NB_API_ALLOWED_ORIGINS` in `.env`.
+    For detailed instructions regarding the query tool see [Running cohort queries](query_tool.md).
+
+    For example, the [`.template-env`](https://github.com/neurobagel/api/blob/main/.template-env) file in the Neurobagel API repo assumes you want to allow API requests from a query tool hosted at a specific port on `localhost` (see the [Docker Compose section](#docker-compose)).
+
+??? example "More examples of `NB_API_ALLOWED_ORIGINS`"
+    ``` bash title=".env"
+    # do not allow requests from any frontend origins
+    NB_API_ALLOWED_ORIGINS=""  # this is the default value that will also be set if the variable is excluded from the .env file
+
+    # allow requests from only one origin
+    NB_API_ALLOWED_ORIGINS="https://query.neurobagel.org"
+
+    # allow requests from 3 different origins
+    NB_API_ALLOWED_ORIGINS="https://query.neurobagel.org https://localhost:3000 http://localhost:3000"
+
+    # allow requests from any origin - use with caution
+    NB_API_ALLOWED_ORIGINS="*"
+    ```
```

Comment:
Maybe add commands for stopping and restarting the API-graph docker-compose network after the `.env` file has been updated?

Reply:
Was that info already here? I agree it'd be helpful, but if it's new then let's not append it to this PR.